pinot-common/src/thrift/request.thrift

```diff
@@ -125,6 +125,7 @@ struct BrokerRequest {
  10: optional string bucketHashKey;
  11: optional bool enableTrace;
  12: optional string responseFormat;
+ 13: optional list<string> routingOptions;
```

Comment:
I thought about that too, so that we could even remove the `enableTrace` flag (or extend it to make it more flexible).

Technically, we could do it in another release (changing the name of the field, not its type) since only the tag is written on the wire.